scripts/scr_chapter_new/scr_chapter_new.gml

```diff
@@ -45,7 +45,7 @@ function scr_chapter_new(argument0) {
 	    selected_chapter=1;chapter=argument0;icon=1;icon_name="da";founding=0;fleet_type=1;strength=10;purity=8;stability=10;cooperation=5;
 	    homeworld="Dead";homeworld_name="The Rock";recruiting_world="Death";recruiting_name="Kimmeria";
 	    homeworld_exists=1;recruiting_exists=1;homeworld_rule=3;aspirant_trial="Survival of the Fittest";
-	    adv[1]="Enemy: Fallen";dis[1]="Never Forgive";
+	    adv[1]="Enemy: Fallen";dis[1]="Never Forgive";doctrine="Breakthrough"
```

Comment:
I don't think the Dark Angels are a "Breakthrough" doctrine adhering chapter.
Unless of course, it is just the Deathwing, with it's terminator and other elite force?
Because if it is with ravenwing, deathwing and... I don't remember what else, it seems more fitting with "Generalist" doctrine.

Reply:
I'll probably switch them to Generalist in the future due to their force composition and fighting style in lore